presto-docs/src/main/sphinx/connector/hive-caching.rst

```diff
@@ -177,16 +177,16 @@ The implementation of the cache exposes a `number of metrics
 :doc:`inspect these and other metrics directly in Presto with the JMX connector
 or in external tools </admin/jmx>`.
 
-The following example query returns the total amount of data fetched by the
-BookKeeper on the coordinator and all the separate nodes on the workers in the
-default async mode:
+If you are using JMX connector, basic caching statistics are exposed via
+``jmx.current."rubix:catalog=<catalog_name>,name=stats"`` table.
+You can read more detailed statistics by querying ``jmx.current."rubix:catalog=<catalog_name>,name=stats``.
+You need to replace ``<catalog_name>`` with name of you Hive catalog.
+
+The following example query returns average cache hit ratio for given Hive catalog.
```

Comment:
The following example query returns the average cache hit ratio for the ``hive`` catalog:

Reply:
A hint @mosabua: 

If you are suggesting rewording of whole paragraphs you can use suggestion mechanism of Github code review. 
To do so:
* mark the lines for which you want to suggest alternative.
* click comment
* click +/- button in the comment editor
* provide suggestion.

Screenshot: 
![image](https://user-images.githubusercontent.com/890111/90536626-97f79180-e17c-11ea-8cd2-310c711cc757.png)